recovery/elliptics_recovery/utils/misc.py

```diff
@@ -178,6 +184,9 @@ def __add__(self, b):
         ret.remove_old_bytes = self.remove_old_bytes + b.remove_old_bytes
         ret.removed_uncommitted_keys = self.removed_uncommitted_keys + b.removed_uncommitted_keys
         ret.merged_indexes = self.merged_indexes + b.merged_indexes
+        ret.skip_write_to_ro_group = self.skip_write_to_ro_group + b.skip_write_to_ro_group
+        ret.skip_remove_corrupted_key_from_ro_group = (self.skip_remove_corrupted_key_from_ro_group +
```

Comment:
https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator